databases/week2/1and3part.sql

```diff
@@ -0,0 +1,66 @@
+use hyf_lesson2;
+SELECT *
+FROM task;
+SELECT*
+from status;
+
+--part 1
+
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id)
+VALUES("shopping", 'by a car', now(), now(),now(),'2');
+
+--Change the title of a task
+UPDATE task
+SET title = 'cancel shopping'
+WHERE id = 36;
+
+--Change a task due date
+update task
+SET due_date = '2022-02-10'
+WHERE id = 36;
+
+--Change a task status
+update task
+SET status_id = '2'
+WHERE id = 36;
+
+--Mark a task as complete
+update task
+SET status_id = '3'
+WHERE id = 36;
+
+--Delete a task
+DELETE FROM task
+WHERE id =36;
+
+--More Queries
+  --part3
+  --Get all the tasks assigned to users whose email ends in @spotify.com
+  use hyf_lesson2;
+  SELECT task.title, user.email
+  FROM task
+  JOIN user_task ON task.id = user_task.task_id
+  JOIN user ON user_task.user_id = user.id
+  WHERE user.email LIKE'%@spotify.com';
+
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+-- somehow it is not showing any output
+use hyf_lesson2;
+SELECT task.title,user.name as user, status.name as status 
```

Comment:
it is correct, its is nice to see you are using Alias, however its nor necessary, just to let you know the VSCode plugin has some bugs and if you dont use alias it may end up with wrong result, but thats not a case in other IDEs or terminal, just to let you know